src/index.html

```diff
@@ -11,7 +11,165 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="fieldset">
+        <legend>Personal Information</legend>
+
+        <label class="person-info">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          Full date of birth:
+          <input
+            type="date"
+            name="bithday"
+          >
```

Comment:
```suggestion
          <input type="date" name="bithday">
```
You can leave in one line, if there are < 3 attributes and the line is not too long